examples/air-quality-sensor-app/silabs/include/AppEvent.h

```diff
@@ -18,39 +18,23 @@
  */
 
 #pragma once
-#include <cstdint>
 
-struct AppEvent;
-typedef void (*EventHandler)(AppEvent *);
+#include "BaseAppEvent.h"
 
-struct AppEvent
+struct AppEvent : public BaseAppEvent
 {
     enum AppEventTypes
     {
-        kEventType_Button = 0,
-        kEventType_Timer,
-        kEventType_AirQualitySensor,
+        kEventType_AirQualitySensor = BaseAppEvent::kEventType_Timer + 1,
```

Comment:
not exactly correct, since if we add more items to `BaseAppEvent::` it might cause collision.

Reply:
Use limits instead of this approach

Reply:
added a kEvent_Max and updated the value based on that